docs/source/api/python/mantid/geometry/DetectorInfo.rst

```diff
@@ -0,0 +1,154 @@
+==============
+ DetectorInfo
+==============
+
+This is a python binding to the C++ class Mantid::Geometry::DetectorInfo.
+
+--------
+Purpose
+--------
+The purpose of the ``DetectorInfo`` object is to allow the user to access information about the detector(s) being used in an experiment. The ``DetectorInfo`` object can be used to access geometric information such as the number of detectors in the beamline, the absolute position of a detector as well as the absolute rotation of a detector.
+
+Many users may need this extra information so that they can have a better understanding of the beamline they are using. This information is easy and fast to access. Some information like mask flags can be modified directly.
+
+The ``DetectorInfo`` object is one of three objects that the user can gain access to from a workspace. 
+The other two are:
+ * ``SpectrumInfo``
+ * ``ComponentInfo``
+
+---------
+Indexing
+---------
+The ``DetectorInfo`` object is accessed by an index going from 0 to N-1, where N is the number of detectors.
```

Comment:
I would recommend to highlight very explicitly and early in the paragraph that the detector index is not the detector ID.